_posts/components/2018-4-1-list-with-icon.html

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "List of items"
+category: components
+description: For having a list of items with an icon in the left side.
+             If this is integrated in body of a card, then add "list-group-item-integrated.border-left-0.border-right-0" 
+             classes for removing duplicate borders to the "list-group-item" div
```

Comment:
of *divs* with the class `list-group-item`

Reply:
do you prefer "elements"